src/System.Transactions/src/System/Transactions/TransactionState.cs

```diff
@@ -1650,13 +1640,10 @@ Guid promoterType
             // Create a durable enlistment.
             Enlistment en = new Enlistment(tx, promotableSinglePhaseNotification, atomicTransaction);
             tx._durableEnlistment = en.InternalEnlistment;
-            if (DiagnosticTrace.Information)
+            TransactionsEtwProvider etwLog = TransactionsEtwProvider.Log;
+            if (etwLog.IsEnabled())
             {
-                EnlistmentTraceRecord.Trace(SR.TraceSourceLtm,
-                    tx._durableEnlistment.EnlistmentTraceId,
-                    EnlistmentType.PromotableSinglePhase,
-                    EnlistmentOptions.None
-                    );
+                etwLog.TransactionstateEnlist(tx._durableEnlistment.EnlistmentTraceId, "PromotableSinglePhase", "None");
```

Comment:
I'd prefer the ETW event method takes the EnlistmentType and EnlistmentOptions enums as parameters and calls ToString() on the values rather than passing in the strings "PromotableSinglePhase" and "None".